eng/tools/compare-packages.ps1

```diff
@@ -0,0 +1,63 @@
+param (
+  $pathToMasterPkg,
+  $pathToCurrentPkg,
+  $dailyDevBuildNo,
+  $pathForDiffFile
+)
+
+function ExtractTGZPackages($pathToPkg){
+  #$regExp = "-\d+\.\d+\.\d+(-\w*\.\d*)?(-dev-$dailyDevBuildNo)?"
+
+  $parentExtractDir = Join-Path $pathToPkg "all-contents"
+  Write-Host "mkdir $parentExtractDir"
+  mkdir $parentExtractDir
+
+  foreach ($p in $(dir $pathToPkg -r -i *.tgz)){
+    $regExp = "(?<name>.*?)(-\d+\.\d+\.\d+.*)"
```

Comment:
nit: I would make this regex string inline where you are using it so it is closer to where you are using it. I'd only pull this out if I planned to share it with other parts of the code.

Reply:
done